src/Objs/Redirect/RedirectObjClass.ts

```diff
@@ -4,5 +4,6 @@ export const Redirect = provideObjClass('Redirect', {
   attributes: {
     title: 'string',
     link: 'link',
+    ensureUserIsLoggedIn: 'boolean',
```

Comment:
* [ ] FYI naming: Looking at other booleans we prefer `require*` for switches like this (we're not fully consistent though). How about `requireLogin` or `loginRequired`?